### DISCUSSION TITLE
How to remove an intelligent suggestion entry?

### DISCUSSION THREAD

**Discussion by Drakemoor** [tadashi-aikawa/obsidian-various-complements-plugin#309] [Q&A]
As the title says.

I have a few misspelled words coming up as suggestions. I do not want to turn off this functionality, just to delete those entries.

Is that possible without directly editing the histories.json file?

**Comment by tadashi-aikawa** [comment]
Hi, @Drakemoor :)

> Is that possible without directly editing the histories.json file?

No, there is no such method.

I don't think such a feature is necessary because the reason misspelled words appear as suggestions is that the misspelling is registered somewhere in the Vault. If you correct the original misspelling, it will no longer appear in the suggestions.
